isolate_example/README.md

```diff
@@ -9,6 +9,27 @@ A sample application that demonstrate best practices when using [`isolates`](htt
 * Demonstrate how to initialize and use an isolate.
 * Show the cost of moving data between isolates and provide alternatives.
 
+## The important bits
+
+### `page_one.dart`
+
+Compares running a large computation on the main isolate vs on a secondary isolate. 
+The `SmoothAnimationWidget` will lower frames when the performance drops.
+
+### `page_two.dart`
+
+An infinite process will be run on the secondary isolate.  The process will be an 
+endless loop of the summation of 100 million randomly generated numbers in increments
+of 10 with the results being printed. The user can start, terminate, pause, resume,
+and change how many time the result will be multiplied by.
+
+### `page_three.dart`
+
+A demonstration of how expensive it is to move large amounts of data between isolates and 
```

Comment:
consider using [active voice](https://developers.google.com/style/voice) e.g. "This sample moves large amounts of data..."

Reply:
+1. For parallelism's sake, I'd recommend starting this off with the verb ("Demonstrates the expense of moving large...")